core/templates/dev/head/pages/exploration_player/exploration_footer_directive.html

```diff
@@ -97,7 +97,10 @@ <h4 translate="I18N_PLAYER_SHARE_THIS_EXPLORATION" class="share-exploration-foot
   }
 
   .oppia-exploration-footer .dropdown-menu.author-profile-dropdown-menu {
-    border-radius: 0;
+    border: none !important;
```

Comment:
Does it work without `!important`? It is best to avoid it if possible because it can easily break things down the road.